modules/gridmap/editor/grid_map_editor_plugin.h

```diff
@@ -187,13 +187,15 @@ class GridMapEditor : public VBoxContainer {
 	Label *info_message = nullptr;
 
 	void update_grid(); // Change which and where the grid is displayed
-	void _draw_grids(const Vector3 &cell_size);
+	void _draw_floor_grid(RID p_grid, int p_floor);
+	void _draw_plane_grid(RID p_grid, Vector3 p_axis_n1, Vector3 p_axis_n2);
```

Comment:
```suggestion
	void _draw_plane_grid(RID p_grid, const Vector3 &p_axis_n1, const Vector3 &p_axis_n2);
```
Pass by reference since you're not modifying 🙂